src/UnitTests/Core/Impl/Threading/TestMainThread.cs

```diff
@@ -8,10 +12,11 @@
 using Microsoft.Common.Core.UI;
 
 namespace Microsoft.UnitTests.Core.Threading {
+    [ExcludeFromCodeCoverage]
     public class TestMainThread : IProgressDialog, IMainThread, IDisposable {
         private readonly Action _onDispose;
         private readonly CancellationTokenSource _cts = new CancellationTokenSource();
-        private readonly AsyncLocal<BlockingLoop> _blockingLoop = new AsyncLocal<BlockingLoop>();
+        private readonly AsyncLocal<BlockingLoop> _blockingLoop = new System.Threading.AsyncLocal<BlockingLoop>();
```

Comment:
Remnant of change where I tried to provide joinable context from here. Will remove.